pkg/cli/cmds/server.go

```diff
@@ -1,19 +1,33 @@
 package cmds
 
 import (
+	"strings"
+
 	"github.com/rancher/k3s/pkg/cli/cmds"
 	"github.com/rancher/rke2/pkg/rke2"
+	"github.com/rancher/wrangler/pkg/slice"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
 const (
-	DisableItems = "rke2-canal, rke2-coredns, rke2-ingress-nginx, rke2-kube-proxy, rke2-metrics-server"
-	rke2Path     = "/var/lib/rancher/rke2"
+	rke2Path = "/var/lib/rancher/rke2"
 )
 
 var (
-	config rke2.Config
+	DisableItems = []string{"rke2-coredns", "rke2-ingress-nginx", "rke2-kube-proxy", "rke2-metrics-server"}
+	CNIItems     = []string{"canal", "cilium"}
+
+	config = rke2.Config{}
+
+	serverFlag = []cli.Flag{
+		&cli.StringFlag{
+			Name:   "cni",
+			Usage:  "(networking) CNI Plugin to deploy, one of none, " + strings.Join(CNIItems, ", "),
```

Comment:
one of none?

Reply:
This is joined with the CNI items list, and uses a common phrasing for selecting one value from a list of valid items:
`one of none, canal, calico`